README.md

```diff
@@ -133,6 +152,10 @@ Topograph offers three endpoints for interacting with the service. Below are the
     "creds": {
       "access_key_id": "id",
       "secret_access_key": "secret"
+    },
+    "params": {
+      "use_simulation": "false",
```

Comment:
mocked_provider?

Reply:
Not used anymore, will be mocked_provider in a future PR.